src/org/omegat/gui/notes/NotesTextArea.java

```diff
@@ -137,12 +135,12 @@ public void redo() {
     public void populatePaneMenu(JPopupMenu menu) {
         final JMenuItem notify = new JCheckBoxMenuItem(OStrings.getString("GUI_NOTESWINDOW_NOTIFICATIONS"));
         notify.setSelected(Preferences.isPreference(Preferences.NOTIFY_NOTES));
-        notify.addActionListener(new ActionListener() {
-            @Override
-            public void actionPerformed(ActionEvent e) {
-                Preferences.setPreference(Preferences.NOTIFY_NOTES, notify.isSelected());
-            }
-        });
+        notify.addActionListener(e -> Preferences.setPreference(Preferences.NOTIFY_NOTES, notify.isSelected()));
         menu.add(notify);
     }
+    
+    @Override
+    public void requestFocus() {
+    	scrollPane.getViewport().getView().requestFocusInWindow();
```

Comment:
We can detect Note pane is hidden or not using a change suggested.

```suggestion
        if (scrollPane.getDockKey().getLocation().equals(DockableState.Location.HIDDEN)) {
            // FIXME: UIDesignManager.getDesktop(this).setAutoHide(<dockable here>, false);
        }
    	scrollPane.getViewport().getView().requestFocusInWindow();
```

I don't know what is `<dockable here>` object. If you find it, we can restore note pane when requested.